app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -578,11 +578,13 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
             val sessionManager = requireComponents.core.sessionManager
             if (session.source == Session.Source.ACTION_VIEW) {
                 sessionManager.remove(session)
+                activity!!.onBackPressed()
             } else {
                 val isLastSession =
                     sessionManager.sessionsOfType(private = session.private).count() == 1
                 if (session.hasParentSession) {
                     sessionManager.remove(session, true)
+                    activity!!.onBackPressed()
```

Comment:
I understand the approach to the other one, but why do we want this one here?

Reply:
For some reason I mistakenly thought it was necessary, but it doesn't actually appear to change anything if you remove it.